orchagent/orchdaemon.cpp

```diff
@@ -120,7 +120,7 @@ bool OrchDaemon::init()
     gDirectory.set(chassis_frontend_orch);
 
     gIntfsOrch = new IntfsOrch(m_applDb, APP_INTF_TABLE_NAME, vrf_orch);
-    gNeighOrch = new NeighOrch(m_applDb, APP_NEIGH_TABLE_NAME, gIntfsOrch);
+    gNeighOrch = new NeighOrch(m_applDb, APP_NEIGH_TABLE_NAME, gIntfsOrch, gFdbOrch, gPortsOrch);
```

Comment:
Looks like we don't need to pass this gPortsOrch. Its not used in this PR. Kindly remove

Reply:
gPortsOrch is used to to get VLAn using bvid. (line 80 of neighorch.cpp). but yes, we do NOT need to attach neighorch to portsorch.